instrumentation/servlet/src/main/java/brave/servlet/WrappedHttpServletRequest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2013-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.servlet;
+
+import brave.Span;
+import brave.http.HttpServerAdapter;
+import brave.http.HttpServerRequest;
+import javax.servlet.http.HttpServletRequest;
+
+/** @since 5.7 */
+// public for others like sparkjava to use
```

Comment:
Instead of public maybe we define a `ServletTracing` class that has factory methods. I was thinking the Runtime class maybe but that isn't public and has a lot of stuff that shouldn't be public

Reply:
actually I don't think there is huge value in sharing this type at all anymore. especially now sparkjava has a route type and we no longer support direct instrumentation with mvc.

I'll hide it.